README.md

```diff
@@ -1 +1,193 @@
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://harry-potter-wiki.netlify.app/)
+- [Loom Video](https://www.loom.com/share/3ca4c5c06ac74a429d90d40abce25b0b)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+-  A web browser like Google Chrome.
+-  A code editor like Visual Studio Code with Git and Node.js.
+
+You can check if Git is installed by running the following command in the terminal.
+```
+$ git --version
+```
+
+Likewise for Node.js and npm for package installation.
+```
+$ node --version && npm --version
+```
+### Setup
+
+Clone this repository using the GitHub link provided below.
+
+
+### Install
+
+In the terminal, go to your file directory and run this command.
+
+```
+$ git clone git@github.com:johnie261/React-capstone.git
+```
+
+
+
+### Usage
+
+
+Kindly modify the files as needed.
+
+In the project directory, you can run:
+```
+$ npm install
+```
+& then
+```
+$ npm run dev
+```
+Runs the app in the development mode.\
+Open [http://localhost:5173](http://localhost:5173) to view it in your browser.
+
+The page will reload when you make changes.
+
+You may also see any lint errors in the console.
+
+
+
+### Run tests
+
+To run tests, run the following command:
+
+```
+$ npm test
+```
+
+
+
+### Deployment
+
+You can deploy this project using:
+
+```
+$ npm run deploy
+```
+
+Or you can use services like Render, Netlify to deploy it. 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Johnie Njoroge**
+
+- GitHub: [@Johnie](https://github.com/johnie261)
+- Twitter: [@Johnie](https://twitter.com/njorogejohnie)
+- LinkedIn: [John Njoroge](https://www.linkedin.com/in/muturijohn/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Original design idea by [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) on Behance.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ]  Customize the App with Animation.
+- [ ]  Add Sliders.
+- [ ]  Customize the Desktop Version.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] it would be better to make sure that the `MIT` link is working and it should take you to the License of the project in order for your project will be professional